tests/admin_changelist/tests.py

```diff
@@ -685,6 +687,29 @@ def test_get_admin_log_templatetag_custom_user(self):
         # it doesn't render any string.
         self.assertEqual(template.render(context), '')
 
+    def test_get_admin_log_templatetag_no_user(self):
+        """
+        Regression test for ticket #24285: get_admin_log template tag doesn't
```

Comment:
"The {% get_admin_log %} tag should work without specifying a user."
(don't really need the ticket number since the behavior is straightforward; the comment about the old implementation isn't import either.